src/Facebook/InstantArticles/Transformer/Rules/Rule.php

```diff
@@ -23,9 +23,8 @@ public function matches($context, $node)
             $log->debug('node matches: '.($matches_node ? 'MATCHES' : 'no match'));
             $log->debug('rule: '.get_class($this));
             $log->debug('-------');
-            return true;
         }
-        if ($node->nodeName === 'img') {
+        if ($node->nodeName === 'iframe') {
```

Comment:
I wont remove this until the logger new feature is delivered: #21 and #14.
We will drop this when those 2 issues are fixed.

So far this is pretty fast and is helping a lot on solving community problems.